package.json

```diff
@@ -80,5 +86,60 @@
       "prettier --write",
       "eslint"
     ]
+  },
+  "standard-version": {
+    "skip": {
+      "tag": true
+    },
+    "types": [
+      {
+        "type": "build",
+        "hidden": true
+      },
+      {
+        "type": "chore",
+        "hidden": true
+      },
+      {
+        "type": "ci",
+        "hidden": true
+      },
+      {
+        "type": "dev",
+        "hidden": true
+      },
+      {
+        "type": "docs",
```

Comment:
I'm on the fence on this one. Can we order the tags or is the order based on the chronological ordering of the commit history? Cause I could see putting `docs` down at the bottom where folks may have lost interest by then or simply not care.

Reply:
I believe the order of sections follows the same order as they're configured here (which I made alphabetical). But I'm fine with doing this & including docs at the end!